Assets/MixedRealityToolkit-Examples/SpatialMapping/Scripts/PlaneFindingTest.cs

```diff
@@ -112,7 +112,7 @@ private void OnDrawGizmosSelected()
                     UnityEditor.Handles.DrawLine(corners[1], corners[2]);
                     UnityEditor.Handles.DrawLine(corners[1], corners[3]);
                     UnityEditor.Handles.DrawLine(corners[2], corners[3]);
-                    UnityEditor.Handles.ArrowHandleCap(0, center, Quaternion.FromToRotation(Vector3.forward, normal), 0.4f, EventType.ignore);
+                    UnityEditor.Handles.ArrowHandleCap(0, center, Quaternion.FromToRotation(Vector3.forward, normal), 0.4f, EventType.Ignore);
```

Comment:
This change should be reverted as well

Reply:
@StephenHodgson I remember that I had the issue on previous Unity version. @johnppella It seems like Unity made an automatic update on this. This gave me error messages on Unity 2017.2

Reply:
As dev is only going to support 2017.2+, I think this should be ok?

Reply:
Auto unity updates should handle these changes. Should leave editor version as is